internal/runner/runner.go

```diff
@@ -75,15 +75,18 @@ func (d *Default) CmdCombinedOutput(cmd *exec.Cmd) ([]byte, error) {
 // Dry is the Runner that only prints commands and does not execute them.
 type Dry struct{}
 
+// CmdRun prints the command.
```

Comment:
Runs?

Reply:
It's dry run, so it's just printing the commands.